bundles/core/src/main/java/com/adobe/cq/wcm/core/components/internal/servlets/AdaptiveImageServlet.java

```diff
@@ -852,7 +849,12 @@ private List<String> selectorToList(String selector) throws IllegalArgumentExcep
         if (StringUtils.isEmpty(selector)) {
             throw new IllegalArgumentException("Expected 1, 2 or 3 selectors instead got empty selector");
         }
-        ArrayList<String> selectorList = Lists.newArrayList(Splitter.on('.').omitEmptyStrings().trimResults().split(selector));
+        ArrayList<String> selectorList = new ArrayList<>();
+        for (String s : selector.split("\\.")) {
+            if (!s.isEmpty()) {
+                selectorList.add(s.trim());
```

Comment:
Wasn't originally the trim before the empty check?

Reply:
Thx, good catch. Fixed it.